wgpu-types/src/lib.rs

```diff
@@ -1993,6 +1998,90 @@ pub enum TextureFormat {
     /// [`Features::TEXTURE_COMPRESSION_ASTC_LDR`] must be enabled to use this texture format.
     #[cfg_attr(feature = "serde", serde(rename = "astc-12x12-unorm-srgb"))]
     Astc12x12RgbaUnormSrgb,
+    /// 4x4 block compressed HDR texture. 16 bytes per block (8 bit/px). Complex pallet. floating-point RGBA.
```

Comment:
It seems to me that we should just encode it more combinatorically:
```rust
Astc { block: AstcBlock, channel: AstcChannel },
...
enum AstcBlock {
  B4x4,
  B5x4,
  ...
}
enum AstcChannel {
  Unorm,
  UnormSrgb,
  Hdr,
}
```
This would allow backends to easily skip Astc, and us to manage the API surface more efficiently

Reply:
This suggestion is so great, now the removed LOC are larger than the added.